prog/plugin/main.go

```diff
@@ -73,26 +73,29 @@ func main() {
 	}
 
 	Log.Println("Weave plugin", version, "Command line options:", os.Args[1:])
+	if noMulticastRoute {
+		Log.Warning("--no-multicast-route option has been removed; multicast is off by default")
+	}
```

Comment:
maybe, but presumably a person who specified `--no-multicast-route` is not the target audience.